psi4/driver/procrouting/proc.py

```diff
@@ -1070,7 +1070,7 @@ def scf_wavefunction_factory(name, ref_wfn, reference, **kwargs):
         wfn.set_basisset("BASIS_RELATIVISTIC", decon_basis)
 
     # Set the multitude of SAD basis sets
-    if (core.get_option("SCF", "GUESS") == "SAD" or core.get_option("SCF", "GUESS") == "HUCKEL"):
+    if (core.get_option("SCF", "GUESS") == "SAD" or core.get_option("SCF", "GUESS") == "SADNO" or core.get_option("SCF", "GUESS") == "HUCKEL"):
```

Comment:
Can we rewrite this to `core.get_option("SCF", "GUESS") in ["SAD", "SADNO", ...]`?

Reply:
Done.